scene/main/node.cpp

```diff
@@ -2080,6 +2134,23 @@ bool Node::is_unique_name_in_owner() const {
 	return data.unique_name_in_owner;
 }
 
+void Node::set_exposed_in_owner(bool p_enabled) {
+	ERR_MAIN_THREAD_GUARD
+	if (data.exposed_in_owner == p_enabled) {
+		return;
+	}
+	data.exposed_in_owner = p_enabled;
+	if (p_enabled) {
+		set_unique_name_in_owner(p_enabled);
+	}
+
+	update_configuration_warnings();
```

Comment:
I see you copied it from `set_unique_name_in_owner()`, but this has no reason to be here.

Reply:
this actually forces the tree to update after removing exposure, otherwise the exposed icon stays in the editor tree.

Reply:
Maybe you should add a comment then.